frontend-wasm/src/code_translator.rs

```diff
@@ -119,6 +119,14 @@ pub fn translate_operator(
         Operator::Call { function_index } => {
             translate_call(state, builder, function_index, mod_info, span, diagnostics)?;
         }
+        Operator::CallIndirect {
```

Comment:
@greenhat Do you know what in dlmalloc is using indirect calls? I'm guessing there must be some kind of configurable behavior that works by providing a function pointer somewhere, but just want to make sure it's not something critical, or we may have to require the use of `wee_alloc`

Reply:
Yes, I linked to `std` and it's `core::fmt` and `std::panicking`. See https://github.com/0xPolygonMiden/miden-ir/commit/4a9e3030e9c323d9ce9ac48c9c0d0504f7313724

Do we intend to require `no_std` in contracts? If so, it's avoidable. 
Also. I have a naive idea of how to implement `call_indirect`. Since we know the func signature at compile time we can filter all the relevant functions from the table and put them in a generated function where calls are dispatched in runtime via `CondBr`s or `If`s.